kafka-ui-react-app/src/theme/theme.ts

```diff
@@ -40,6 +41,13 @@ export const Colors = {
     '30': '#90caf9',
     '40': '#64b5f6',
   },
+  gray: {
+    '10': '#f9fafa',
+    '20': '#ebebeb',
+    '30': '#e7e7e7',
+    '40': '#e6e6e6',
+    '80': '#919497',
+  },
```

Comment:
Do you take these colors from figma? 
Oleg told me that we should take the colors from figma.

Reply:
i got these colors from their rgba versions for add they to Theme